components/_patterns/02-molecules/date-bars/_date-bar.scss

```diff
@@ -0,0 +1,23 @@
+.date-bar {
+  align-items: center;
+  background-color: $color-success;
+  color: $white;
+  display: flex;
```

Comment:
I prefer as much as possible to handle layout at the "block" level.

So, in this case `.date-bar` is the block, and `.date-bar__date` is the element.

Right now, the element is set to `flex: 0 0 100%;` in order to make the layout work correctly (the items should be stacked instead of side-by side). If I were writing this, I'd tell the parent to stack the items, space them evenly, and align them to the left.

Here's what that would look like:
```
.date-bar {
  display: flex;
  flex-flow: column nowrap;
  justify-content: space-around;
  align-items: flex-start;
}
```

With that, you don't have to add any layout styles to the element.